pkg/otlp/rum/mapping.go

```diff
@@ -0,0 +1,307 @@
+	Datadog + "." + View + "." + Error + "." + Count:                                         View + "." + Error + "." + Count,
+	Datadog + "." + View + "." + Crash + "." + Count:                                         View + "." + Crash + "." + Count,
+	Datadog + "." + View + "." + LongTask + "." + Count:                                      View + "." + LongTask + "." + Count,
+	Datadog + "." + View + "." + FrozenFrame + "." + Count:                                   View + "." + FrozenFrame + "." + Count,
+	Datadog + "." + View + "." + SlowFrames:                                                  View + "." + SlowFrames,
+	Datadog + "." + View + "." + SlowFrames + "." + StartTime:                                View + "." + SlowFrames + "." + StartTime,
+	Datadog + "." + View + "." + SlowFrames + "." + Duration:                                 View + "." + SlowFrames + "." + Duration,
+	Datadog + "." + View + "." + Resource + "." + Count:                                      View + "." + Resource + "." + Count,
+	Datadog + "." + View + "." + Frustration + "." + Count:                                   View + "." + Frustration + "." + Count,
+	Datadog + "." + View + "." + InForegroundPeriods:                                         View + "." + InForegroundPeriods,
+	Datadog + "." + View + "." + InForegroundPeriods + "." + StartTime:                       View + "." + InForegroundPeriods + "." + StartTime,
+	Datadog + "." + View + "." + InForegroundPeriods + "." + Duration:                        View + "." + InForegroundPeriods + "." + Duration,
+	Datadog + "." + View + "." + MemoryAverage:                                               View + "." + MemoryAverage,
+	Datadog + "." + View + "." + MemoryMax:                                                   View + "." + MemoryMax,
+	Datadog + "." + View + "." + CPUTicksCount:                                               View + "." + CPUTicksCount,
+	Datadog + "." + View + "." + CPUTicksPerSecond:                                           View + "." + CPUTicksPerSecond,
+	Datadog + "." + View + "." + RefreshRateAverage:                                          View + "." + RefreshRateAverage,
+	Datadog + "." + View + "." + RefreshRateMin:                                              View + "." + RefreshRateMin,
+	Datadog + "." + View + "." + SlowFramesRate:                                              View + "." + SlowFramesRate,
+	Datadog + "." + View + "." + FreezeRate:                                                  View + "." + FreezeRate,
+	Datadog + "." + View + "." + FlutterBuildTime + "." + Min:                                View + "." + FlutterBuildTime + "." + Min,
+	Datadog + "." + View + "." + FlutterBuildTime + "." + Max:                                View + "." + FlutterBuildTime + "." + Max,
+	Datadog + "." + View + "." + FlutterBuildTime + "." + Average:                            View + "." + FlutterBuildTime + "." + Average,
```

Comment:
Am i reading this rt - the only diff is in prefix ? if so can we just add/remove prefix rather than this map

Reply:
This file scares me. it would be hard to maintain. If say for 90% of them it is prefix based, lets remove this from here do a prefix based mapping.